spec/unit/appmgmt_spec.rb

```diff
@@ -49,42 +49,152 @@ def compile_to_env_catalog(string)
         cons { two: host => ahost, consume => Cap[cap] }
```

Comment:
@lutter This Cons resource ends up not having $host set in the environment catalog. I can see it evaluating this part `"cons { two: consume => Cap[cap] }"` which parser says comes from here.